addon/-private/system/model/errors.js

```diff
@@ -95,7 +95,35 @@ var MapWithDefault = Ember.MapWithDefault;
     </div>
   {{/each}}
   ```
-
+  
+  The JSON API spec also allows for object level errors to be placed
+  in an object with pointer `data`
+  
```

Comment:
Looks like travis didn't like this pr because of extra trailing whitespace on some of these blank lines. @adam-knights do you mind amending this commit and removing the whitespace?